src/api/index.ts

```diff
@@ -24,66 +30,53 @@ type Options = {
    * Path to use for requests via the proxy, defaults to ''
    */
   proxyPathBase?: string;
-  configApiRef: ConfigApi
+  configApiRef: ConfigApi;
 };
 //@ts-ignore
 export const snykApiRef: ApiRef<SnykApi> = createApiRef<SnykApi>({
   id: "plugin.snyk.service",
 });
 
-
 export interface SnykApi {
   ListAllAggregatedIssues(orgName: string, projectId: string): Promise<any>;
-  ListAllAggregatedLicenseIssues(
-    orgName: string,
-    projectId: string
-  ): Promise<any>;
-  ListAllIgnoredIssues(orgName: string, projectId: string): Promise<any>;
   ProjectDetails(orgName: string, projectId: string): Promise<any>;
-  ProjectList(orgName: string): Promise<any>;
+  ProjectsList(orgName: string, repoName:string): Promise<any>;
```

Comment:
what data is needed from this? There are some filters and fields that are only available in an older Projects REST API atm

Reply:
projects listing got switched to REST APIs. Project details is still on v1 currently. will be replaced at a later time.